### PR TITLE
docs(all): update initial user docs.

### PR DESCRIPTION
**Pull request by damienwebdev**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/graycoreio/daffodil/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?
The docs don't do the repo justice.

Fixes: N/A

## What is the new behavior?
This clarifies the initial ux of arriving on the daffodil repo.

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,81 +1,55 @@
...
+[![dependencies Status](https://david-dm.org/graycoreio/daffodil/status.svg)](https://david-dm.org/graycoreio/daffodil)
+[![devDependencies Status](https://david-dm.org/graycoreio/daffodil/dev-status.svg)](https://david-dm.org/graycoreio/daffodil?type=dev)
 
 ## What is Daffodil?
-Daffodil is a set of frontend libraries that intends to accomplish three things:
-1. Improve the developer workflow when writing frontend software for online stores.
+Daffodil is a frontend toolkit that intends to accomplish three things:
+
+1. Improve developer workflow when writing frontend software for online stores.
 2. Provide a consistent frontend developer tool-chain regardless of a business's chosen Ecommerce platform
```
Comment (xelaint) [inline README.md]: We're using lowercase `ecommerce`

File: docs/INSTALLATION.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+# Installation
+This will walk you through a simple
```
Comment (xelaint) [inline docs/INSTALLATION.md]: a simple what?

File: docs/INSTALLATION.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
+# Installation
+This will walk you through a simple 
+
+## Prerequisites
+
+1. [Node v8+](https://nodejs.org/en/)
+2. [Angular CLI](https://cli.angular.io/)
+
+## Setup Your Angular Project
```
Comment (xelaint) [inline docs/INSTALLATION.md]: Set Up

File: docs/INSTALLATION.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    * `npm install --save @daffodil/checkout` - Install Checkout Module
+
+## Install Peer Dependencies
+
+1. `npm install --save @ngrx/store @ngrx/effects @ngrx/entity`
+
+## Configure Your App
+1. Add the following to your application's root `AppModule`.
+
+```ts
```
Comment (xelaint) [inline docs/INSTALLATION.md]: Remove ts

Comment (damienwebdev) [reply docs/INSTALLATION.md]: @xelaint the `ts` marks the codeblock for typescript formatting.

File: README.md (hunk 1 above)
Comment (damienwebdev) [reply README.md]: Fixed by aadf67e4

File: docs/INSTALLATION.md (hunk 3 above)
Comment (damienwebdev) [reply docs/INSTALLATION.md]: Fixed by bc578275103e38834dc62dd38d81f1a6f4f22fd6

File: docs/INSTALLATION.md (hunk 2 above)
Comment (damienwebdev) [reply docs/INSTALLATION.md]: resolved in db6d1ff9f835e6982b243ff053f6bf6321dd0e88

File: docs/INSTALLATION.md [hunk 5]
```diff
@@ -41,9 +45,11 @@ ng new my-store
 export class AppModule { }
 ```
 
-2. Replace the defaults with your store's url in your `AppModule`
-3. `ng serve`
-3. Visit http://localhost:4200 in your browser
+> Be sure to Replace the defaults with your store's url in your `AppModule`
```
Comment (xelaint) [inline docs/INSTALLATION.md]: `replace`

Comment (damienwebdev) [reply docs/INSTALLATION.md]: resolved in `d2987766643fb817985c3a9c679558d14a3754f1`

**Comment by lderrickable** [comment]
lgtm

**Pull request merged**